inc/service_structs.h

```diff
@@ -126,9 +126,10 @@ typedef enum
 {
     // Luos specific registers
     RTB_CMD = ROBUS_PROTOCOL_NB, // Ask(size == 0), generate(size == 2), or share(size > 2) a routing_table.
-    WRITE_ALIAS,                 // Get and save a new given alias.
-    UPDATE_PUB,                  // Ask to update a sensor value each time duration to the sender
-    NODE_UUID,                   // luos_uuid_t
+    RTB_CMD_GLOBAL,
```

Comment:
RTB the representation of the entire system (so it's global). The specific one is the local RTB.
RTB_CMD should be LOCAL_RTB
RTB_CMD_GLOBAL should be RTB
Also don't forget to update comments on those 2 commands.